dataset-spec/template.schema.json

```diff
@@ -116,18 +172,24 @@
                         "required": [
                             "name",
                             "long_name",
-                            "units",
-                            "valid_min",
-                            "valid_max",
-                            "time_range",
                             "dtype",
                             "dims",
-                            "sources"
+                            "units"
```

Comment:
Units should not be required or allow for `null` value. Categorical data has no defined units (not even unitless).

Reply:
OK, updated in both schema and spec.